introduction.md

```diff
@@ -3,4 +3,7 @@ DanTheMan Life Rules
 - if anything is fixed then make it better or break it
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

Reply:
good idea